lib/psulib_traject/workers/indexer.rb

```diff
@@ -6,10 +6,10 @@ class Indexer < Base
       # @param [Hash] args
       # @option args [String, Pathname] :filename (required)
       # @option args [String] :collection_name
-      def perform(filename, opts = {})
+      def perform(filename, collection_name = nil)
```

Comment:
was there a reason the hash parameter wouldn't work here? It's better to have optional parameters as a hash so you don't have to worry about positionality later on.

Reply:
no idea, when called from rake the argument gets ignored. 

`bundle exec rake traject:index['foo','bar']`  bar gets ignored.

Reply:
I'd still like to have an opts hash here. Which test was failing?

Reply:
OK, this is fixed now, but the rake tasks needed to be updated to reflect the different arguments.

Reply:
I misspoke, _shocker_ i know. The issue was rake sends the argument as a string, and rspec sends the argument as a symbol, so when the tests were passing the method was ignoring the argument that the rake test was passing, i guess the rake task could do something like args.collection.to_sym? or something. feels hacky

Reply:
Yeah, I just updated the rake tasks with the new positional arguments.